tests/functional/contribute/test_friends.py

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.contribute.friends import ContributeFriendsPage
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_display_signup_form(base_url, selenium):
```

Comment:
Basket ID still isn't in basket prod, and this is supposed to go live tomorrow, so filed [bug 1261078](https://bugzilla.mozilla.org/show_bug.cgi?id=1261078) to add tests after merge.